Sources/GRPCCodeGen/Internal/StructuredSwiftRepresentation.swift

```diff
@@ -66,11 +65,14 @@ struct ImportDescription: Equatable, Codable {
 /// A description of an access modifier.
 ///
 /// For example: `public`.
-enum AccessModifier: String, Equatable, Codable {
+public enum AccessModifier: String, Sendable, Equatable, Codable {
```

Comment:
We should keep this `internal`

Reply:
Should all of the structs and enums from this file be internal?

Reply:
They should be no more visible than `internal`, fine if some are `fileprivate`/`private`. There shouldn't be any `public` types though.